meta/main.yml

```
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: MIT
+---
+galaxy_info:
+  author: Radovan Sroka <rsroka@redhat.com>
+  description: Fapolicyd system role
+  company: Red Hat Inc.
+  license: MIT
+
+  min_ansible_version: "2.9"
+  platforms:
+    - name: Fedora
+      versions:
+        - all
+    - name: EL
+      versions:
+        - 8
+        - 9
```

```suggestion
        - "9"
```
sorry - I forgot that all version numbers must be strings